tests/utils.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const waitMs = (n) => new Promise((resolve) => {
+	setTimeout(() => {
+		resolve();
+	}, n);
+});
```

Comment:
Can we move the method to `tests/helpers/utils.js` as we already have `utils.js` file for tests
```suggestion
const waitMs = (n) => new Promise((resolve) => {
	setTimeout(() => {
		resolve();
	}, n);
});
```

Reply:
applied.